ci/test_linux.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+#
+# Test Snowflake SQLAlchemy in Linux
+# NOTES:
+#   - Versions to be tested should be passed in as the first argument, e.g: "3.7 3.8". If omitted 3.7-3.10 will be assumed.
+#   - This script assumes that ../dist/repaired_wheels has the wheel(s) built for all versions to be tested
+#   - This is the script that test_docker.sh runs inside of the docker container
+
+PYTHON_VERSIONS="${1:-3.7 3.8 3.9 3.10}"
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SQLALCHEMY_DIR="$( dirname "${THIS_DIR}")"
+
+# Install one copy of tox
+python3 -m pip install -U tox tox-external-wheels
```

Comment:
We've moved SQLAlchemy to use tox4, so `tox-external-wheels` is not necessary
```suggestion
python3 -m pip install --pre tox
```